### ISSUE TITLE
Eco Led not working properly after GUI upgrade.

### ISSUE THREAD

**Issue by lorenzocanale-LC** [Ansuel/tch-nginx-gui#890]
Device Model: DGA4132 FW: 2.2.0 
Gui Version: 9.5.14

Description of problem: When upgrading GUI with ECO Led on, the ECO Led stays on but the wireless led stays on (blinking with net traffic). Solution is to turn off and then on again ECO Led option

**Comment by FrancYescO** [comment]
duplicate of https://github.com/Ansuel/tch-nginx-gui/issues/799 ?

**Comment by lorenzocanale-LC** [comment]
I don't think so, it happens only after a gui upgrade to me

**Comment by FrancYescO** [comment]
do you use wifi nsc ?

**Comment by lorenzocanale-LC** [comment]
It is on in the gui, but I never changed it. I'll try to turn it off

**Comment by FrancYescO** [comment]
ON is the default value

**Comment by lorenzocanale-LC** [comment]
I just tried with NSC off, the led starts blinks again after the gui upgrade. (It's only the 2.4Ghz)

**Comment by marcotini** [comment]
> I just tried with NSC off, the led starts blinks again after the gui upgrade. (It's only the 2.4Ghz)

Yeah me too, only the 2.4 is off.
